src/test/java/teammates/test/cases/webapi/PublishFeedbackSessionActionTest.java

```diff
@@ -23,13 +30,92 @@ protected String getRequestMethod() {
     @Test
     @Override
     protected void testExecute() throws Exception {
-        // TODO
+        ______TS("Typical case");
+
+        CourseAttributes course = typicalBundle.courses.get("typicalCourse1");
+        FeedbackSessionAttributes session = typicalBundle.feedbackSessions.get("session1InCourse1");
+
+        String[] params = {
+                Const.ParamsNames.COURSE_ID, course.getId(),
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, session.getFeedbackSessionName(),
+        };
+
+        PublishFeedbackSessionAction publishFeedbackSessionAction = getAction(params);
+
+        JsonResult result = getJsonResult(publishFeedbackSessionAction);
+        FeedbackSessionData feedbackSessionData = (FeedbackSessionData) result.getOutput();
+
+        assertEquals(feedbackSessionData.getFeedbackSessionName(), session.getFeedbackSessionName());
```

Comment:
The first parameter if assertEquals should be expected value not actual value. Check other places and other PR also :P

Reply:
Updated